react/features/base/participants/reducer.js

```diff
@@ -53,6 +54,10 @@ const PARTICIPANT_PROPS_TO_OMIT_WHEN_UPDATE
  */
 function _participant(state: Object = {}, action) {
     switch (action.type) {
+    case CONFERENCE_WILL_JOIN:
```

Comment:
I'm afraid I don't find `CONFERENCE_WILL_JOIN` to be an acceptable action type though because it may very well arrive after I've been able to tap on a thumbnail.

Reply:
I don't think that's possible.

Reply:
I'll look further for an alternative.

Reply:
You mean it's not possible because we no longer as of yesterday evening render the filmstrip before a second participant joins? (If it's that, then I don't approve of such a dependency.) Or something else?